source/installation/preparation.rst

```diff
@@ -109,12 +115,12 @@ L'équivalent pour Nginx serait :
 
        location ~ \.php$ {
            include snippets/fastcgi-php.conf;
-           fastcgi_pass unix:/var/run/php7.0-fpm.sock;
+           fastcgi_pass unix:/var/run/php/php7.0-fpm.sock;
```

Comment:
on my Fedora, path is `/var/run/php-fpm/www.sock`. Maybe add a comment to tell the user he have to adapt the path according to its php installation?

Reply:
Alright. Actually, a user who would just copy/paste the configuration would have some issue. So yeah, I rollback all of this and place the comment as you suggest.